src/main/java/edu/ie3/datamodel/models/input/system/EmInput.java

```diff
@@ -0,0 +1,213 @@
+/*
+ * © 2022. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.datamodel.models.input.system;
+
+import edu.ie3.datamodel.models.ControlStrategy;
+import edu.ie3.datamodel.models.EmControlStrategy;
+import edu.ie3.datamodel.models.OperationTime;
+import edu.ie3.datamodel.models.input.NodeInput;
+import edu.ie3.datamodel.models.input.OperatorInput;
+import edu.ie3.datamodel.models.input.system.characteristic.ReactivePowerCharacteristic;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.UUID;
+
+public class EmInput extends SystemParticipantInput {
+
+  /** Reference via UUID to all SystemParticipantInputs connected to this model */
+  private final UUID[] connectedAssets;
+
+  /** Reference to the control strategy to be used for this model */
+  private final ControlStrategy controlStrategy;
+  /**
+   * Constructor for an operated energy management system
+   *
+   * @param uuid of the input entity
+   * @param id of the asset
+   * @param operator of the asset
+   * @param operationTime time for which the entity is operated
+   * @param node the asset is connected to
+   * @param qCharacteristics description of a reactive power characteristic
+   * @param connectedAssets array of all connected assets
+   * @param controlStrategy control strategy used for this model
+   */
+  public EmInput(
+      UUID uuid,
+      String id,
+      OperatorInput operator,
+      OperationTime operationTime,
+      NodeInput node,
+      ReactivePowerCharacteristic qCharacteristics,
+      UUID[] connectedAssets,
```

Comment:
I don't know how much effort it would take in PSDM, but I think it'd be nice to have the connected assets here directly instead of just their UUIDs. There's some discussion to be had here for sure, and this might not be possible in the short term though.

Reply:
Created issue #640